pulsar/internal/lookup_service.go

```diff
@@ -30,8 +30,9 @@ import (
 
 // LookupResult encapsulates a struct for lookup a request, containing two parts: LogicalAddr, PhysicalAddr.
 type LookupResult struct {
-	LogicalAddr  *url.URL
-	PhysicalAddr *url.URL
+	IsProxyThroughServiceURL bool
```

Comment:
It seems `IsProxyThroughServiceURL` is never used?

Reply:
You're right, it's never used outside of `lookupService.GetBrokerAddress`, where it is retrieved and consumed at the same time. Removed, thanks!